examples/src/main/scala/org/apache/spark/examples/ml/BucketedRandomProjectionLSHExample.scala

```diff
@@ -21,9 +21,9 @@ package org.apache.spark.examples.ml
 // $example on$
 import org.apache.spark.ml.feature.BucketedRandomProjectionLSH
 import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.sql.SparkSession
```

Comment:
It seems to be in a few examples; but it's not consistent, and this keeps the code cleaner, even if the import is redundant on a spark-shell session.